openmaptiles/postserve.py

```diff
@@ -148,12 +123,22 @@ def serve(port, pghost, pgport, dbname, user, password, metadata, tileset_path,
         (
             r"/tiles/([0-9]+)/([0-9]+)/([0-9]+).pbf",
             GetTile,
-            dict(fname=fname, connection=connection, query=query)
+            dict(pool=pool)
         ),
     ])
 
     application.listen(port)
     print(f"Postserve started, listening on 0.0.0.0:{port}")
     print(f"Use http://localhost:{port} as the data source")
-
     tornado.ioloop.IOLoop.instance().start()
+
+
+def m():
```

Comment:
this file is not the entry point - we use [bin/postserve](https://github.com/openmaptiles/openmaptiles-tools/blob/master/bin/postserve) for that, and I don't think we should have two entry points for the same program.